api/processors/auxil/log_event_creator.py

```diff
@@ -7,14 +9,20 @@
 )
 
 
+class FromOrToGroup(enum.Enum):
+    FROM = "from_group"
+    TO = "to_group"
+
+
 class GroupLogEventCreator:
     @staticmethod
-    def create(
+    def create(  # noqa: PLR0913
         group: Group,
         group_log_event_type: GroupLogEventType,
         student_log_event_type: StudentLogEventType,
         teacher_log_event_type: TeacherLogEventType,
         coordinator_log_event_type: CoordinatorLogEventType | None = None,
+        from_or_to_group: FromOrToGroup = FromOrToGroup.FROM,
```

Comment:
Is the default value justified by the fact that the most views we have already done create log events with `group_from` rather than `group_to`? If so, I think that it should be marked as a temporary solution with a TODO and then a separate PR should be prepared where this default value is removed and all views set this arg explicitly. There is no general logic behind this default value, so leaving it like this could make the code confusing.

Also, when a group is created (#174), the log event should have a `group_to`, not `group_from` attribute set. I guess this is a mistake I didn't spot then.

Reply:
Now it supports both. And also fixed issue with group creation.